helloCampusService.js

```diff
@@ -231,6 +233,28 @@ function createQuestion(req, res, next) {
         });
 }
 
+function deleteQuestion(req, res, next) {
+    db.one(
+"DELETE FROM Question WHERE ID = ${id}", req.params)
```

Comment:
I'd recommend this, same for the other one

```suggestion
function deleteQuestion(req, res, next) {
    db.one(
"DELETE FROM Question WHERE ID = ${id} RETURNING ID", req.params)
```